supervisor.go

```diff
@@ -142,8 +142,8 @@ func (s *SupervisorService) SuspendAll() (string, error) {
 // Resume calls druid Supervisor service's Resume API.
 // https://druid.apache.org/docs/latest/api-reference/supervisor-api/#resume-a-supervisor
 func (s *SupervisorService) Resume(supervisorID string) (OutputIngestionSpec, error) {
-	r, err := s.client.NewRequest(http.MethodPost, applySupervisorId(supervisorResumeEndpoint, supervisorID), nil)
 	var result OutputIngestionSpec
+	r, err := s.client.NewRequest(http.MethodPost, applySupervisorId(supervisorResumeEndpoint, supervisorID), nil)
```

Comment:
We should avoid naked params.
https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters

```suggestion
	r, err := s.client.NewRequest(http.MethodPost, applySupervisorId(supervisorResumeEndpoint, supervisorID), nil /*no data*/)
```